src/AdminConsole/Migrations/Sqlite/20231024113214_BillingRefactoring.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Passwordless.AdminConsole.Migrations.Sqlite;
+
+/// <inheritdoc />
+public partial class BillingRefactoring : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.AddColumn<string>(
+            name: "BillingPlan",
+            table: "Organizations",
+            type: "TEXT",
+            nullable: false,
+            defaultValue: "");
+
+        migrationBuilder.AddColumn<string>(
+            name: "BillingSubscriptionItemId",
+            table: "Organizations",
+            type: "TEXT",
+            nullable: true);
+
+        migrationBuilder.Sql("UPDATE Organizations SET BillingPlan = a.BillingPlan, BillingSubscriptionItemId = a.BillingSubscriptionItemId FROM Organizations o INNER JOIN Applications a ON o.Id = a.OrganizationId;");
+
+        migrationBuilder.DropColumn(
```

Comment:
Breaking database change - not something we want to do. We're only "allowed" to add columns, since we must be able to apply migrations but then roll back to previous version of code, in case of bugs etc.